witchcraft-logging-formatting/src/test/java/com/palantir/witchcraft/java/logging/format/TraceLogFormatterTest.java

```diff
@@ -43,4 +43,24 @@ void formats_correctly() {
                 .isEqualTo(
                         "[2019-12-25T01:02:03Z] traceId: abdefghijklmno id: id name: name duration: 31 microseconds");
     }
+
+    @Test
+    void formats_tags_correctly() {
+        String formatted = TraceLogFormatter.format(TraceLogV1.builder()
+                .type("trace.1")
+                .time(TestData.XMAS_2019)
+                .span(Span.builder()
+                        .traceId("abdefghijklmno")
+                        .id("id")
+                        .name("name")
+                        .timestamp(SafeLong.of(999))
+                        .duration(SafeLong.of(31))
+                        .tags("tagName", "tagValue")
+                        .build())
+                .build());
+
+        assertThat(formatted)
+                .isEqualTo("[2019-12-25T01:02:03Z] traceId: abdefghijklmno id: id name: "
```

Comment:
nit: I find this a bit hard to read as there is no good separation between the span attributes. Maybe adding a comma or some other divider might help a little?
`[2019-12-25T01:02:03Z] traceId: abdefghijklmno, id: id, name: name, duration 31 ms`

Reply:
Good idea -- I'd originally implemented these identically to the golang formatters, however we've since taken a few liberties and we don't get any value out of keeping them identical.

I've always found it confusing that there's a `traceId` and `id` field. I think I'll rename those to `trace` and `span`. We probably don't need a title descriptor for the span name, and I can replace ` microseconds` with `µs` e.g. `150 microseconds` becomes `150µs`

Reply:
Sounds good to me. Also fine to do this in a separate PR if you want to get the tags out.

Reply:
I couldn't get the `name: ` title removal to feel natural, so I've kept it in, but taken care of the other changes :-)

Reply:
Should be ready to merge if you're happy with it.

Reply:
Looks good!